clang/lib/Headers/yvals_core.h

```diff
@@ -0,0 +1,26 @@
+/* ===-------- vadefs.h ---------------------------------------------------===
+ *
+ * Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+ * See https://llvm.org/LICENSE.txt for license information.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ *===-----------------------------------------------------------------------===
+ */
+
+/* Only include this if we are aiming for MSVC compatibility. */

```

Comment:
Newer lib/Headers changes prefer `//`-style comments. Do we need `/* */`?

Reply:
We do not need `/* */`.

I was just copying from the other files in this folder for the copyright header and clang-format didn't complain.
I'll give the style guide a read and get this fixed :).